tensorhive/models/User.py

```diff
@@ -111,37 +114,23 @@ def find_by_username(cls, username):
         else:
             return result
 
-    @hybrid_property
-    def as_dict(self):
+    def as_dict(self, include_private=False, include_groups=True):
         """
         Serializes current instance into dict.
+        :param include_private: passed to CRUDModel as_dict
+        :param include_groups: flag determining if user groups should be included (False to prevent recurrence)
         :return: Dictionary representing current instance.
         """
-        return self._as_dict(True)
-
-    @hybrid_property
-    def as_dict_shallow(self):
-        """
-        Serializes current instance into dict. Will not include user's groups (to prevent recurrence).
-        :return: Dictionary representing current instance (without groups).
-        """
-        return self._as_dict(False)
+        user = super(User, self).as_dict(include_private)
 
-    def _as_dict(self, include_groups):
         try:
             roles = self.role_names
         except Exception:
             roles = []
         finally:
-            user = {
-                'id': self.id,
-                'username': self.username,
-                'createdAt': DateUtils.stringify_datetime(self.created_at),
```

Comment:
Same situation with date format as above.

Reply:
fixed in 1a64c0bc9706b0c7d4ca36227f9c6840b84abe03